app/pb_kits/playbook/pb_table/docs/_table_sm.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= pb_rails("table", props: {size: "sm"}) do %>
+<%= pb_rails("table", props: { size: "sm", container: false }) do %>
```

Comment:
It's picky, but you used a space after opening and before closing the curly brackets here, but did not do that in the other .erb files.

Reply:
Fixed the spacing in all html files, thanks!